### DISCUSSION TITLE
Why &OPENLIB instead of &CURLIB when executing defualt actions?

### DISCUSSION THREAD

**Discussion by angelorpa** [codefori/vscode-ibmi#1199] [Q&A]
hi @worksofliam ,  why default actions like CRTBNDRPG use the library where the source member resides and not the current working library set in the config (&CURLIB) ? Doesn't it make more sense to use the current library?

![image](https://user-images.githubusercontent.com/10833316/228627146-879e2171-bc31-4615-8874-ef2cdce01cc3.png)

**Comment by chrjorgensen** [comment]
@angelorpa The default actions can all be changed to suit your needs - so you can change the commands to use `&CURLIB` instead of `&OPENLIB`. The defaults are only suggestions. Many users create their own build / compile commands.

**Comment by worksofliam** [comment]
Oh my! Sorry I let this slip!

I originally used `&OPENLIB` for everything because I have met way too many users who have QGPL as their default current library.

As Christian mentioned, it is easy to change so users can set it to how they want - as I'm sure you know!
